src/cace/ari/base.h

```diff
@@ -359,6 +359,13 @@ int ari_init_move(ari_t *ari, ari_t *src);
  */
 int ari_deinit(ari_t *ari);
 
+/** Reset an initialized ARI to the undefined value.
```

Comment:
**Suggestion**: Also mention allocated memory will be freed, since we are doing  a `deinit`.